HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml

```diff
@@ -1206,6 +1224,7 @@
       <sch:assert role='ERROR' test='count(h:FractionCoolLoadServed) = 1'>Expected 1 element(s) for xpath: FractionCoolLoadServed</sch:assert>
       <sch:assert role='ERROR' test='count(h:AnnualCoolingEfficiency[h:Units="EER"]/h:Value) = 1'>Expected 1 element(s) for xpath: AnnualCoolingEfficiency[Units="EER"]/Value</sch:assert>
       <sch:assert role='ERROR' test='count(h:AnnualHeatingEfficiency[h:Units="COP"]/h:Value) = 1'>Expected 1 element(s) for xpath: AnnualHeatingEfficiency[Units="COP"]/Value</sch:assert>
+      <sch:assert role='ERROR' test='count(h:AttachedToGeothermalLoop) &lt;= 1'>Expected 0 or 1 element(s) for xpath: AttachedToGeothermalLoop</sch:assert>
```

Comment:
Can you add a test in test_validation.rb for a bad idref? Like [this](https://github.com/NREL/OpenStudio-HPXML/blob/319f860c55f4bc56a21a1c119a6c365a29169f2d/HPXMLtoOpenStudio/tests/test_validation.rb#L929).

Reply:
How does this relate to `'orphaned-geothermal-loop'`? If I add the suggested test above, I see:
```
Found extra error messages:
["C:/OpenStudio/OpenStudio-HPXML/workflow/sample_files/tmp.xml: Geothermal loop 'GeothermalLoop1' found but no heat pump attached to it."]
```

Reply:
I just ran base-hvac-ground-to-air-heat-pump-detailed.xml and changed AttachedToGeothermalLoop idref to "foo". I got both error messages:

```
Error: Attached geothermal loop 'foo' not found for heat pump 'HeatPump1'.
Error: Geothermal loop 'GeothermalLoop1' found but no heat pump attached to it.
```

That's probably what will happen most often. But technically you could trigger the first error message without the second. For example, you could have a GSHP with AttachedToGeothermalLoop="GeothermalLoop1" but there is no GeothermalLoop1 in the file. Or you could have two GSHPs, where one has a valid AttachedToGeothermalLoop idref and one has an invalid AttachedToGeothermalLoop idref (so there's no orphaned geothermal loop).